pallets/grants/src/mock.rs

```diff
@@ -95,32 +95,49 @@ impl pallet_balances::Config for Test {
 }
 
 ord_parameter_types! {
+	pub const ALICE: AccountId = 1;
+	pub const BOB: AccountId = 2;
 	pub const CancelOrigin: AccountId = 42;
 	pub const ForceOrigin: AccountId = 43;
 }
 
+parameter_types! {
+	pub static MaxSchedule: u32 = 2;
+}
+
 impl Config for Test {
 	type Event = Event;
 	type Currency = PalletBalances;
 	type CancelOrigin = EnsureSignedBy<CancelOrigin, AccountId>;
 	type ForceOrigin = EnsureSignedBy<ForceOrigin, AccountId>;
+	type MaxSchedule = MaxSchedule;
 	type WeightInfo = ();
 	type BlockNumberProvider = frame_system::Pallet<Test>;
 }
 
-pub const ALICE: AccountId = 1;
-pub const BOB: AccountId = 2;
+pub(crate) fn balances(who: &AccountId) -> (Balance, Balance) {
```

Comment:
```suggestion
pub fn balances(who: &AccountId) -> (Balance, Balance) {
```

Reply:
Any specific reason ? its utility function scope is restricted only to unit test context, and not going to be used outside the pallet.

Reply:
Since we haven't exported them out in a public module, `pub` could be behaving the same as `pub(crate)` and was the shorter less verbose syntax. However on a second thought going for a lower level visibility as a habit could be a good choice.